tst/EnergyPlus/unit/Fixtures/IdfParserFixture.hh

```diff
@@ -0,0 +1,83 @@
+#ifndef IdfParserFixture_hh_INCLUDED
+#define IdfParserFixture_hh_INCLUDED
+
+// Google Test Headers
+#include <gtest/gtest.h>
+
+// EnergyPlus Headers
+#include <EnergyPlus/DataStringGlobals.hh>
+
+#include "../TestHelpers/IdfParser.hh"
+
+#include <ostream>
+
+namespace EnergyPlus {
+
+	class IdfParserFixture : public testing::Test
+	{
+	protected:
+		static void SetUpTestCase() { }
+		static void TearDownTestCase() { }
+
+		virtual void SetUp() { }
+
+		virtual void TearDown() { }
+
+		// This function creates a string based on a vector of string inputs that is delimited by DataStringGlobals::NL by default, but any 
+		// delimiter can be passed in to this funciton. This allows for cross platform output string comparisons.
+		std::string delimited_string( std::vector<std::string> const & strings, std::string const & delimiter = DataStringGlobals::NL ) {
+			std::ostringstream compare_text;
+			for( auto const & str : strings ) {
+				compare_text << str << delimiter;
+			}
+			return compare_text.str();
+		}
+
+		void eat_whitespace( std::string const & idf, size_t & index ) {
+			IdfParser parser;
+			parser.eat_whitespace( idf, index );
```

Comment:
I think I originally had it as static methods. I don't know why I changed it. Free functions could work as well. Let me think about it some.